app/views/assignments/_form.html.erb

```diff
@@ -304,27 +304,42 @@
                         '.form_description') %>
       </p>
       <div id='grace_periods'>
+        <h2><%= t('submission_rules.grace_period_submission_rule.short_name') %></h2>
+        <table id='grace_periods_table'>
+          <thead>
+          <tr>
+            <th><%= label :hours,
```

Comment:
The HTML semantics might be a little misleading. The column header is not a label for any one particular input tag, so I don't think "label" is actually appropriate here.